### ISSUE TITLE
Old documentation for Python version 0.12.0 is still accessible

### ISSUE THREAD

**Issue by sherlockbeard** [delta-io/delta-rs#2805]
# Environment

**Delta-rs version**:

**Binding**: 

**Environment**:
- **Cloud provider**:
- **OS**:
- **Other**:

***
# Bug
Go to https://delta-io.github.io/delta-rs/python/installation.html you will see documentation for version 0.12.0

**What happened**:

**What you expected to happen**:

Only latest version documentation should be present .

**How to reproduce it**:

**More details**:

**Comment by omkar-foss** [comment]
Yes! It's still accessible and comes up in top search engine results for "delta-rs docs". I stumble upon it every once in a while 😁

Probably we can consider a 301 redirect from the old to the new docs to maintain SEO and then eventually decommission the old docs.

**Comment by ion-elgreco** [comment]
We should I guess just remove it, we did a redirect on the main page, I think it's fine to remove the old ones after so long.

**Comment by omkar-foss** [comment]
Yes the [old docs home page](https://delta-io.github.io/delta-rs/python/index.html) properly redirects to the new docs. But some other old docs pages (e.g. [API Reference](https://delta-io.github.io/delta-rs/python/api_reference.html)) still show up in top search results as they don't have a redirect.

>We should I guess just remove it, we did a redirect on the main page, I think it's fine to remove the old ones after so long.

Indeed, I agree! The best way would be to remove old docs altogether to avoid any sort of confusion for users.

**Comment by ion-elgreco** [comment]
Old docs have been removed. Also the new docs is versioned!
